liquidity/liquidity.go

```diff
@@ -902,7 +913,10 @@ func (m *Manager) checkExistingAutoLoops(ctx context.Context,
 				out.Contract.MaxMinerFee,
 				mSatToSatoshis(prepay.Value),
 			)
-		} else if !out.LastUpdateTime().Before(m.params.AutoFeeStartDate) {
+		} else if out.LastUpdateTime().After(
```

Comment:
Doesn't this mean that even if a swap was started in the last budget period, it will be counted for the current period?

Reply:
Yes, if a swap is still active in any way within our current period. regardless of when it started, it will count in the current period's budget.